common/web/types/src/util/util.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * xml2js will not place single-entry objects into arrays. Easiest way to fix
+ * this is to box them ourselves as needed. Ensures that o.x is an array.
+ *
+ * @param o Object with property to box
+ * @param x Name of element to box
+ */
```

Comment:
while we're "going there" this might be a future pr.

Reply:
It's not a generic solution, which would need much more work. It's specific to our format at this time. xml2js is a bit rough for my liking, mostly because of the file format mismatches...